.github/workflows/pr-contract-check.yml

```
@@ -0,0 +1,135 @@
+      - name: Calculate code hash
+        id: codehash
+        working-directory: ${{ matrix.contract }}
+        run: |
+          # Use b2sum (blake2) to calculate hash
+          HASH=$(b2sum -l 256 target/ink/*.wasm | cut -d' ' -f1)
+          echo "hash=$HASH" >> $GITHUB_OUTPUT
+          echo "Contract: ${{ matrix.contract }}"
+          echo "Code Hash: $HASH"
+          
+      - name: Post results as PR comment
+        uses: actions/github-script@v6
```

actionlint (1.7.7)

the runner of "actions/github-script@v6" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)